index.html

```diff
@@ -52,4 +52,184 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
       </ul>
     </div>
   </section>
+  <section>
+    <div class="grid-container">
+      <div class="img-container">
+        <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
+      </div>
+      <div class="card">
+        <div class="card-element">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
```

Comment:
- [ ] Please, it is better to wrap one card in one container instead of letting inner card elements be unwrapped as you did, this can be helpful for your styling and HTML best practice.
by the way, your image container is overlapping, see this image to have an idea of what is getting wrong and fix it, please.
<img width="318" alt="t" src="https://user-images.githubusercontent.com/79058364/153677187-f3a93686-4158-470e-b0b8-e321887833e1.png">

#### wrapping those 2 divs in one content will make things easier, don't you think so? let's go and fix it then

- [ ] Consider `font-size, margin, padding and font-family` too, your card should look like this, kindly take this one into consideration and fix it
- [ ]  Figma template

<img width="413" alt="fig" src="https://user-images.githubusercontent.com/79058364/153676803-2993ee22-19a2-4bc2-b175-2c0f895309df.png">

Reply:
Thanks for the suggestion. I have considered your suggestion. In my case, I have designed it for a mobile version i.e 375 px and for that the design is working fine. I will make the changes as you suggested further in the Desktop version.

Reply:
This issue is not resolved

Reply:
@Kingstalux @Kingstalux 
I have considered your suggestion. In my case, I have designed it for a mobile version i.e 375 px and for that the design is working fine. I will make the changes as you suggested further in the Desktop version.
![Review](https://user-images.githubusercontent.com/52527576/153697467-93bf1199-7f03-493f-ae3e-f298653d6c13.png)

Reply:
Hello @thenarain I see your site looks good on your machine but in web development, the site should look good in all machines. It would be best to place the project images inside the same `div` tag as the rest of the project information so that no matter the screen size, it will still look good 👍

Reply:
Thanks for the valuable suggestion. I have implemented all the changes.